tests/integration/test_model.py

```diff
@@ -924,3 +924,79 @@ def test_experiment_automl_regressor(c, client, training_df):
     ), "Best model was not registered"
 
     check_trained_model(c, "my_automl_exp2")
+
+
+# TODO - many ML tests fail on clusters without sklearn - can we avoid this?
```

Comment:
@ayushdg , What will be the recommended way to test something that has a install dependency ?